math/perfectnumber_test.go

```diff
@@ -0,0 +1,157 @@
+package math_test
+
+import (
+	"testing"
+
+	"github.com/TheAlgorithms/Go/math"
+)
+
+type testCaseSumOfProperDivisors struct {
+	input    uint
+	expected uint
+}
+
+// getSumOfProperDivisorsTestCases returns an array of example data
+// for the tests of the function SumOfProperDivisors.
+// Example data was verified using [A001065].
+// [A001065]: https://oeis.org/A001065/b001065.txt
+func getSumOfProperDivisorsTestCases() []testCaseSumOfProperDivisors {
```

Comment:
There is no testCase for the bad path...

Reply:
I really do not see a bad path in the function `SumOfProperDivisors(inNumber uint) uint` (integer overflow possibly?). Could you please explain in more detail what do you have in mind?

Reply:
Nevermind, I didn't see the test cases properly. The `{0, 0}` and `{1, 0}` are the bad cases in my mind.